content/en/docs/setup/getting-started/index.md

```diff
@@ -79,6 +79,12 @@ Follow these steps to get started with Istio:
     selected to have a good set of defaults for testing, but there are other
     profiles for production or performance testing.
 
+    {{< warning >}}
+    If your platform has a custom demonstration profile, e.g., Openshift, use
```

Comment:
We call the profiles different things in different places: ex: Some additional vendor-specific configuration profiles are also available. For a complete list, refer to Vendor Optimized Profiles.